flow/enhancers/graph.go

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
This file has been deleted in a previous patch, need to be removed

Reply:
good point ... fixed